packages/terra-core-docs/src/terra-dev-site/doc/alert/AccessibilityGuide.2.doc.mdx

```diff
@@ -42,6 +43,36 @@ Terra Alert has an optional role prop but should be used carefully when the defa
 | Alert | See <Hyperlink href="https://w3c.github.io/aria/#alert" variant="external">WAI-ARIA Alert Role</Hyperlink>[<sup>[1]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region with important, and usually time-sensitive, information." |
 | Status | See <Hyperlink href="https://w3c.github.io/aria/#status" variant="external">WAI-ARIA Status Role</Hyperlink>[<sup>[1]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region whose content is advisory information for the user but is not important enough to justify an alert, often but not necessarily presented as a status bar." |
 
+<Notice variant="important" ariaLevel="5">
+
+#### Accessibility Guidance: Alert Notification Type
```

Comment:
Same with these. I don't think we need to call anything out specifically as Accessibility Guidance within an Accessibility guide, so pretty much all of the content on this page within "Important" blocks can be placed within the normal flow of the page.